backend/data/defineTransactionHashesInSql.ts

```diff
@@ -0,0 +1,25 @@
+import fs from 'node:fs';
+
+import { getRowsOfExpectedOutput } from './csvToJson.js';
+
+const transactionHashesFilename = './backend/data/transactionHashes.sql';
+const transactionHashesCsv = 'data/transactionHashes.csv';

```

Comment:
I would rename this to `TRANSACTION_HASHES_FILENAME` and `TRANSACTION_HASHES_CSV` since they are hardcoded values

Reply:
I'll do that now.